ansible/roles/stack-sunbird/templates/ml-analytics-service.conf

```diff
@@ -79,15 +79,15 @@ batch_rollup_url = http://{{groups['druid'][0]}}:8081/druid/indexer/v1/task
 
 urlQuery = http://{{groups['druid'][0]}}:8082/druid/v2?pretty
 
-datasource_url = http://{{groups['ml-raw-broker'][0]}}:8082/druid/v2/datasources
+datasource_url = http://{{groups['ml-raw-broker'][0]}}:{{ml_raw_broker_port}}/druid/v2/datasources
```

Comment:
@Vivek-M-08  Why do you want to variablise the port number? where are you initialising the new variables with default values?

Reply:
@gandham-santhosh - As Druid can be deployed in cluster mode with different services having different IPs and ports, these values will be sourced from the private repository of the adopters. Our scripts will use these values to check the status of Druid and its services.

CC: @aks30